cucumber-rails.gemspec

```diff
@@ -22,10 +22,10 @@ Gem::Specification.new do |s|
   }
 
   s.add_runtime_dependency('capybara', ['>= 2.12', '< 4'])
-  s.add_runtime_dependency('cucumber', ['>= 3.0.2', '< 4'])
+  s.add_runtime_dependency('cucumber', ['4.0.0.rc3'])
```

Comment:
@luke-hill AFAIK the version is `4.0.0.rc.3`. And might as well bump it to rc.4 while you at it. 👍